reference/SimpleCalculator/SimpleCalculator.Tests/SimpleCalculator.Tests.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>net6.0</TargetFramework>
+		<IsPackable>false</IsPackable>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<PackageReference Include="coverlet.collector" Version="3.1.0" />
+		<PackageReference Include="FluentAssertions" Version="6.7.0" />
+		<PackageReference Include="Microsoft.Extensions.Logging" Version="7.0.0" />
+		<PackageReference Include="Microsoft.Extensions.Logging.Console" Version="7.0.0" />
+		<PackageReference Include="Microsoft.Net.Compilers.Toolset" Version="4.2.0">
+		  <PrivateAssets>all</PrivateAssets>
+		  <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+		</PackageReference>
+		<PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.3.0" />
+		<PackageReference Include="NUnit" Version="3.13.2" />
+		<PackageReference Include="NUnit3TestAdapter" Version="4.0.0" />
+		<PackageReference Include="Uno.Extensions.Logging.WebAssembly.Console" Version="1.4.0" />
+		<PackageReference Include="Uno.Extensions.Reactive.WinUI" Version="2.3.0-dev.318" />
+		<PackageReference Include="Uno.Material.WinUI" Version="2.4.0-dev.64" />
+		<PackageReference Include="Uno.Material.WinUI.Markup" Version="2.4.0-dev.64" />
+		<PackageReference Include="Uno.Toolkit.WinUI.Markup" Version="2.4.0-dev.115" />
+		<PackageReference Include="Uno.UI.Adapter.Microsoft.Extensions.Logging" Version="4.7.0-dev.266" />
+		<PackageReference Include="Uno.WinUI.Markup" Version="4.6.0-dev.13" />
+		<PackageReference Include="Uno.WinUI.RemoteControl" Version="4.7.0-dev.266" />
+		<PackageReference Include="Uno.WinUI.WebAssembly" Version="4.7.0-dev.266" />
```

Comment:
Anything Uno related in here should be removed, as it's not going to be available in a net6.0 test project.